app/views/layouts/board.html.erb

```diff
@@ -15,13 +15,34 @@
   <li role="presentation"><%= link_to('Users', users_path) %></li>
 </ul>
 
-<table>
-  <tr>
-    <td id="board">
+<div class="container-fluid">
+  <div class="row">
+    <div id="board" class="col-md-8">
       <%= yield %>
-    </td>
-  </tr>
-</table>
+    </div>
+    <div class="col-md-2">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">Color Key</h3>
+        </div>
+        <ul class="list-group">
+          <li class="list-group-item">
+            <div class="color-box green"></div>
```

Comment:
What about using badges instead so that it's native to bootstrap and CSS doesn't have to be maintained.  http://getbootstrap.com/components/#badges